doc/nrf/releases/release-notes-1.7.0.rst

```diff
@@ -0,0 +1,447 @@
+.. _ncs_release_notes_changelog:
+
+|NCS| v1.6.0 Release Notes
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products.
+It includes the TF-M, MCUboot and the Zephyr RTOS open source projects, which are continuously integrated and re-distributed with the SDK.
+
+The |NCS| is where you begin building low-power wireless applications with Nordic Semiconductor nRF52, nRF53, and nRF91 Series devices.
+nRF53 Series devices are now supported for production.
+Wireless protocol stacks and libraries may indicate support for development or support for production for different series of devices based on verification and certification status.
```

Comment:
Hi, @jdstapes , do these lines need to be updated? Checking as per https://projecttools.nordicsemi.no/jira/browse/NCSDK-11021 -- "Check/update the development/production support statement". Thanks!

Reply:
Checked at the doc review meeting. No changes needed.